web-app/js/portal/filter/NumberFilter.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2015 IMOS
+ *
+ * The AODN/IMOS Portal is distributed under the terms of the GNU General Public License
+ *
+ */
+
+Ext.namespace('Portal.filter');
+
+Portal.filter.NumberFilter = Ext.extend(Portal.filter.Filter, {
+
+    getSupportedGeoserverTypes: function() {
+
+        return ['double', 'float', 'integer', 'int', 'long', 'short', 'decimal'];
+    },
+
+    getUiComponentClass: function() {
+
+        return Portal.filter.ui.NumberFilterPanel;
+    },
+
+    getCql: function() {
+
+        return this._getCql();
+    },
+
+    getHumanReadableForm: function() {
+
+        return this._getCql(this.getLabel());
+    },
+
+    _getCql: function(alternateLabel) {
+
+        var cql = String.format(
+            '{0} {1} {2}',
+            alternateLabel || this.getName(),
+            this._getOperator(),
```

Comment:
should we attempt to convert the operator into a human readable form here? For instance, >= becomes 'greater than or equal to'